meetings/working-groups/discriminated-unions/conceptual-overview.md

```diff
@@ -0,0 +1,55 @@
+# Conceptual overview
+
+> The text that follows is the working draft for how we'll introduce this concept to customers. It's meant to address two distinct challenges:
+>
+> - Many of the scenarios for unions can also be handled by inheritance. That prompts the question on the value of adding unions to C#.
+>
+> - Unions have different meanings in different computer languages. There are tagged unions, type unions, C-style (unsafe) unions. Which do we mean?
+>
+> The language is intended to be less formal than the specification and serve as an introduction for customers.
+
+*Discriminated unions* provide a new way to express the concept that *an expression may be one of a fixed set of types or values*.
```

Comment:
Does this imply exhaustiveness? Have we settled on being exhaustive (it's fine with me, but I think Cyrus had some Roslyn scenarios for open)